.env.sample

```diff
@@ -0,0 +1,4 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_DB=POSTGRES_DB
+POSTGRES_USER=POSTGRES_USER
+POSTGRES_PASSWORD=POSTGRES_PASSWORD
```

Comment:
What is the reason to have 2 identical samples of the .env file? 
Also, there is no reason to explicitly define  env_file in docker-compose.yml. You can leave it as it is just having the .env file in your directory.
Look at the reference
https://docs.docker.com/compose/environment-variables/
![image](https://user-images.githubusercontent.com/95351527/209431667-d300f836-6128-43ab-85ad-811818f1e93f.png)